### PR TITLE
Prevent null killers from breaking scoreboard.

### PR DESCRIPTION
**Pull request by sysrage**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/CUModSquad/cu-web-serverstats/pull/12" target="_blank">ReviewNinja

File: src/stores/kills.js [hunk 1]
```diff
@@ -88,10 +88,12 @@ var Kills = Reflux.createStore({
 
         // count kills and deaths per player
         for (var i = 0; i < kills.length; i++) {
-            var k = kills[i].killer, v = kills[i].victim;
-            if (k.id !== v.id) {        // ignore suicides
-                (players[k.name] = players[k.name] || { player: k, kills: [], deaths: [] }).kills.push(kills[i]);
-                (players[v.name] = players[v.name] || { player: v, kills: [], deaths: [] }).deaths.push(kills[i]);
+            if (kills[i].killer && kills[i].victim) {   // ignore entries with a null killer
```
Comment (saddieeiddas) [inline src/stores/kills.js]: looks good, makes sense to check if killer and victim are truthy before adding them to kills/deaths

**Comment by saddieeiddas** [comment]
looks good, makes sense to check if killer/victim are truthy before adding them, although it would be good to know why there are `null` records here?

File: src/stores/kills.js (hunk 1 above)
Comment (Mehuge) [inline src/stores/kills.js]: The if should go after var k = ... and be simply if (k && v) could even combine it with the expression on line 93 so we have  if (k &&v && k.id != v.id)

**Comment by sysrage** [comment]
I've made the changes suggested by Mehuge.

**Comment by sysrage** [comment]
Git fail on my part...

**Pull request closed without merging**